build_examples.sh

```diff
@@ -4,3 +4,4 @@ set -eux
 
 cargo build --release --target=thumbv7em-none-eabi --examples
 cargo build --release --target=riscv32imc-unknown-none-elf --examples # Important for tests: This target does not support atomics
+cargo build --release --target=riscv32imac-unknown-none-elf --examples
```

Comment:
This will make the build chain significantly slower. I would prefer to not test the risv32imac target explicitly as the `cfg(target = "riscv32")` compile switch is already tested via the riscv32imc target.

Reply:
Fine with me!